Lib/test/test_ctypes/test_aligned_structures.py

```diff
@@ -38,16 +29,16 @@ class Main(base):
             main = Main.from_buffer(data)
             self.assertEqual(main.first, 7)
             self.assertEqual(main.string.value, b'hello world!')
-            self.assertEqual(
-                bytes(main.string.__buffer__(inspect.BufferFlags.SIMPLE)),
-                b'\x68\x65\x6c\x6c\x6f\x20\x77\x6f\x72\x6c\x64\x21\x00\x00\x00\x00'
-            )
+            self.assertEqual(bytes(main.string), b'hello world!\0\0\0\0')
             self.assertEqual(Main.string.offset, 16)
+            self.assertEqual(Main.string.size, 16)
```

Comment:
It is not affected by the `_align_` setting, because its size was already 16. Maybe make the `value` field type `c_char * 12`? Then the `_align_` setting will affect also the size of `Aligned`.